CMakeLists.txt

```diff
@@ -717,6 +717,7 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/widget/wstarrating.cpp
```

Comment:
Please check that the first line of your commit message is short enough to avoid line wrapping

Reply:
Okay. I will take care of that and add long messages as commit paragraphs in new lines.
By the way, what is the preferred syntax for commit messages? I was kinda going with "feature/file: the change".

Reply:
We don't have a set format. We just recently had a [Zulip discussion](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/commit.20message.20format) about this that didn't really reach a conclusion. "feature/file: the change" works well, but including both "feature" and "file" takes up a lot of characters. I do "ClassName: the change" instead of listing the file path.